api/lib/mno_enterprise/concerns/controllers/jpi/v1/impac/dashboards_controller.rb

```diff
@@ -28,15 +28,10 @@ def show
   # POST /mnoe/jpi/v1/impac/dashboards
   #   -> POST /api/mnoe/v1/users/1/dashboards
   def create
-    # TODO: dashboards.build breaks as dashboard.organization_ids returns nil, instead of an
-    #       empty array. (see MnoEnterprise::Impac::Dashboard #organizations)
-    # @dashboard = dashboards.build(dashboard_create_params)
-    # TODO: enable authorization
-    # authorize! :manage_dashboard, @dashboard
-    # if @dashboard.save
+    authorize! :create_impac_dashboards, dashboards.build(dashboard_create_params)
```

Comment:
we're doing a double instantiation here.
Could we have?
```ruby
@dashboard = dashboards.build(dashboard_create_params)
authorize! :create_impac_dashboards, @dashboard 
if @dashboard.save
  ...
end
```
Or is it not working as the old commented code is saying?

Reply:
Indeed it doesn't work as `build + save` does not seem to behave the same as `create`: some parameters (as the owner_id for example) are missing when the `save` Her-built request is sent to mno-hub, as a result, the record is not persisted correctly.

I've tried several things but always run into the same issues so I think we'd better let this code unchanged for now.